docs/kit/Adoption View/00_adoption-view.md

```diff
@@ -31,13 +62,106 @@ The main difference between the EDC and the previous connectors of the [IDSA][id
 - The connection of software systems on the customer and provider side
 
+## Additional Resources
+
+### Catena-X Standards
+
+The Connector KIT builds on the [Catena-X Standards][Catena-X-Standards-url]. Every data consumer and provider in Catena-X is required to comply with these standards. The [EDC Connector][edc-url] builds on the following standards:
+
+- [CX - 0002 Digital Twins in Catena - X][CX0002]
+- [CX - 0003 BAMM Aspect Meta Model][CX0003]
+- [CX - 0008 Relevant standards for conformity assessments][CX0008]
+- [CX - 0018 Eclipse Data Space Connector (EDC)][CX0018]
+- [CX - 0044 ECLASS][CX044]
+- [CX - 0047 DCM Data Model Material Demand & Capacity Group][CX0047]
+- [CX - 0059 Triangle Behavioral Twin Endurance Predictor][CX0059]
+
+### Terminology
+
+![domain-model](images/domain-model.png)
+
+> The shown picture illustrates only a generic view of the Domain Model and is not intended to show all aspects of the project.
+
+#### Extensions
+
+There are different extenions for the Connector, e.g. for the Data Plane. This enables various tranfer modes like httpData or via blob-storage.
+
+#### Data Plane
+
+The Data Plane handles several forms of actual data exchange by utilizing various extensions.
+
+#### Control Plane
+
+The Control Plane handles meta data exchange with other components and Connectors, as well as transfer of access tokens
+
+#### Data Assets
+
+Data Sources (databases, files, cache information, etc.) are connected to the Connector and are represented by Data Assets. For each asset, a [`DataAddress`](#data-address) needs to be resolvable.
+
+#### Data address
+
+A data address is a pointer into the physical storage location where an asset will be stored.
+
+#### Policy Definition
+
+A standardized set of policies can be used to define actions regarding access to and usage of assets. These actions can be limited further by constraints (temporal or spatial) and duties ("e.g. deletion of the data after 30 days").
+
+#### Contract Definition
+
+By combining [`Assets`](#data asset) and Policies, Contracts for data offerings are defined. These Contracts need to be accepted by consuming participants (Connectors) for the data exchange to take place.
+
+#### Contract offer
+
+The contract offer is a representation of the [`ContractDefinition`](#contract-definition) for a specific consumer and serves as protocol for a data transfer object (DTO) for a particular contract negotiation. If a data consumer wants to conclude a binding data exchange contract based on the terms of a Data Offer, the data consumer can communicate such desire to the data provider by referencing to a specific Data Offer. Under German law, this constitutes a binding offer by the data consumer. For now, the dtaa consumer only has the option to accept all terms of a Data Offer (or not). The Data Exchange Process does not yet provide for the data consumer to make an offer that deviates from the terms of a Data Offer as set by the data provider.
```

Comment:
I don't know whether the reference to German law is appropriate here, as the EDC hopes to be international in scope.